src/index.ts

```diff
@@ -91,3 +99,5 @@ export type {
   Steps,
   actionTitle,
 } from './formatter/javascript';
+
+export { log } from './core/logger';
```

Comment:
likewise, this is a helper purely for the library as it uses the internals to identify if we have to log or not. I dont like the idea of exposing it and allowing users to rely on this internal details.

Reply:
Removed.